modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc

```diff
@@ -210,9 +210,9 @@ image::shared:byoc_apply.png[cloud_byoc_apply]
 
 Redpanda Cloud does not support the following self-hosted functionality:
 
-- OpenID Connect (OIDC) authentication
+- Data transforms (currently in beta for Redpanda Cloud)
+- OIDC authentication
 - Kerberos authentication
-- Data transforms
 - Remote Read Replicas
```

Comment:
RRR - should show (Currently in beta for Redpanda Cloud)

Reply:
fixed